input/inputmanager.h

```diff
@@ -6,13 +6,13 @@
 #include "controllable.h"
 #include "producer.h"
 
-#include "qmlinputdevice.h"
-#include "sdleventloop.h"
-#include "inputdevice.h"
-#include "keyboard.h"
-
 #include "logging.h"
 
+class InputDevice;
```

Comment:
I can see you doing your forward declarations here. For consistency's sake, we should use my style.